quarkus-opentelemetry-exporter-azure/deployment/src/main/java/io/quarkiverse/opentelemetry/exporter/azure/deployment/AzureExporterAlwaysEnabledProcessor.java

```diff
@@ -0,0 +1,15 @@
+package io.quarkiverse.opentelemetry.exporter.azure.deployment;
+
+import io.quarkus.deployment.Feature;
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.deployment.builditem.FeatureBuildItem;
+
+// Executed even if the extension is disabled, see https://github.com/quarkusio/quarkus/pull/26966/
+public class AzureExporterAlwaysEnabledProcessor {
+
+    @BuildStep
+    FeatureBuildItem feature() {
+        return new FeatureBuildItem(Feature.OPENTELEMETRY_JAEGER_EXPORTER);
```

Comment:
jaeger?

Reply:
It's because of legacy reasons

Reply:
We don't need this FeatureBuildItem at all. We can skip the method.